cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowSpecification.java

```diff
@@ -79,6 +81,18 @@ public WorkflowSpecification(String className, String name, String description,
     return nodeIdMap;
   }
 
+  public boolean containsSparkProgram() {
```

Comment:
Since this class is in API, would it be better to add a method called ``containsProgram(SchedulableProgramType type)`` instead of tying it to spark specifically?